### PR TITLE
Add pyfizi and fizi program

### PR DESCRIPTION
**Pull request by quattro**

### CODE REVIEW

File: recipes/pyfizi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  commands:
+    - fizi --help
+
+about:
+  home: https://github.com/bogdanlab/fizi
+  license: GNU General Public v3 (GPLv3)
+  license_family: GPL3
+  license_file:
+  summary: Impute GWAS summary statistics using reference genotype data
+  doc_url:
```
Comment (xhochy) [inline recipes/pyfizi/meta.yaml]: Please omit empty fields.

File: recipes/pyfizi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - pyfizi
+  commands:
+    - fizi --help
+
+about:
+  home: https://github.com/bogdanlab/fizi
+  license: GNU General Public v3 (GPLv3)
+  license_family: GPL3
+  license_file:
```
Comment (xhochy) [inline recipes/pyfizi/meta.yaml]: This is required, either the package should come with one or you need to add the license to the PR.

File: recipes/pyfizi/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a4b47cf057fce63e6d912f87d0e497065436f4b501698543863b5c82970c56f6
+
+build:
+  skip: True  # [win or py!=36]
```
Comment (xhochy) [inline recipes/pyfizi/meta.yaml]: Is this package available for 2.7 and 3.7 and just not 3.6? (just curious)

Comment (quattro) [reply recipes/pyfizi/meta.yaml]: It's available only for 3.6. I wanted to support py >= 36, but the conda environment wouldn't resolve under 37 due to pandas-plink.

File: recipes/pyfizi/meta.yaml (hunk 2 above)
Comment (quattro) [reply recipes/pyfizi/meta.yaml]: I added 'LICENSE' to the conda recipe; I had this earlier but the build failed. There is a LICENSE in the github repo, so I'm not sure if that is how to best link to it.

File: recipes/pyfizi/meta.yaml (hunk 3 above)
Comment (xhochy) [reply recipes/pyfizi/meta.yaml]: Ah, I misread the statement :D 
Here's a PR to add Python 3.7 support for `pandas-plink`: https://github.com/conda-forge/pandas-plink-feedstock/pull/32

Comment (quattro) [reply recipes/pyfizi/meta.yaml]: Thanks for the help!

File: recipes/pyfizi/meta.yaml (hunk 2 above)
Comment (chrisburr) [reply recipes/pyfizi/meta.yaml]: @quattro (It doesn't prevent this from being merged but just so you know)

Technically the files on PyPI are in violation of the licence as they don't include a copy. You can include it automatically in future release tarballs on PyPI if you edit your `setup.cfg` to include:

```
[metadata]
license_files = LICENSE
```

See [here (and the sample project)](https://packaging.python.org/guides/distributing-packages-using-setuptools/#license-txt) for more details.

Comment (quattro) [reply recipes/pyfizi/meta.yaml]: Great thanks. I'll add it to the repo and make sure its included in the next official release. Cheers.

File: recipes/pyfizi/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.14.5
```
Comment (chrisburr) [inline recipes/pyfizi/meta.yaml]: I think only `python` and `pip` are needed in the `host` section. (For a Python package, this only needs to contain dependencies which are actually used by `setup.py`.)

Comment (quattro) [reply recipes/pyfizi/meta.yaml]: Okay I'll retry with those removed. I had generated the initial script with `conda skeleton pyfizi` and edited from there.

**Pull request merged**